### PR TITLE
convert score to percentage, wrap matches in links

### PR DESCRIPTION
**Pull request by susiekims**
I converted the score to percentages for both Company and matches.
While I was there I fixed a small bug in how i was pluralizing match(es), and i wrapped the card in a link to the company

image
image

### CODE REVIEW

File: src/app/components/Company.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ class Company extends React.Component {
 
   render() {
     const { companyData, checklist } = this.props;
+    const scorePercentage = Math.round((companyData.score / 13) * 100);
```
Comment (Kikisek) [inline src/app/components/Company.js]: Do we have access to the `frontendData` in this component? Maybe we could divide the score by the length of the frontendData....I am not sure if that would make sense though

File: src/app/components/Student.js [hunk 2]
```diff
@@ -55,10 +56,12 @@ class Student extends React.Component {
         </div>
         <Typography variant="overline" gutterBottom>
           {studentMatches.length} Match
-          {studentMatches.length > 1 && studentMatches.length !== 0 ? 'es' : null}
+          {studentMatches.length > 1 || studentMatches.length === 0 ? 'es' : null}
```
Comment (Kikisek) [inline src/app/components/Student.js:60]: Nicely done! 👍

File: src/app/components/Student.js [hunk 3]
```diff
@@ -55,10 +56,12 @@ class Student extends React.Component {
...
         </Typography>
         {studentMatches.map(match => (
-          <Match key={match.id} match={match} />
+          <Link to={`/companies/${match.id}`} key={match.id}>
```
Comment (Kikisek) [inline src/app/components/Student.js:63]: This is genius! Love this 😍

**Review by Kikisek — DISMISSED** [review]
Great job! 🥇

File: src/app/components/Company.js (hunk 1 above)
Comment (susiekims) [reply src/app/components/Company.js]: Good idea! In case we ever change the number of checklist items

**Review by Kikisek — APPROVED** [review]
Perfect! Thank you for changing it this fast 💯

**Pull request merged**